src/main/app/src/app/gebruikersvoorkeuren/zoekopdracht-save-dialog/zoekopdracht-save-dialog.component.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+import {GebruikersvoorkeurenService} from '../gebruikersvoorkeuren.service';
+import {Zoekopdracht} from '../model/zoekopdracht';
+import {FormControl} from '@angular/forms';
+import {Werklijst} from '../model/werklijst';
+import {Observable} from 'rxjs';
+import {map, startWith} from 'rxjs/operators';
+import {UtilService} from '../../core/service/util.service';
+
+@Component({
+    selector: 'zac-zoekopdracht-save-dialog',
```

Comment:
Zover ik kan zien wordt deze selector nergens gebruikt. Haal die dan maar weg want dat is meteen een indicatie dat dit component nooit op die manier gebruikt wordt.